packages/react-native-reanimated/src/ConfigHelper.ts

```diff
@@ -2,57 +2,23 @@
 import type { LoggerConfig } from 'react-native-worklets';
 import { updateLoggerConfig } from 'react-native-worklets';
 
-import { ReanimatedError, SHOULD_BE_USE_WEB } from './common';
-import { executeOnUIRuntimeSync, jsiConfigureProps } from './core';
-import { PropsAllowlists } from './propsAllowlists';
-
-function assertNoOverlapInLists() {
-  for (const key in PropsAllowlists.NATIVE_THREAD_PROPS_WHITELIST) {
-    if (key in PropsAllowlists.UI_THREAD_PROPS_WHITELIST) {
-      throw new ReanimatedError(
-        `Property \`${key}\` was whitelisted both as UI and native prop. Please remove it from one of the lists.`
-      );
-    }
-  }
-}
-
-export function configureProps(): void {
-  assertNoOverlapInLists();
-  jsiConfigureProps(
-    Object.keys(PropsAllowlists.UI_THREAD_PROPS_WHITELIST),
-    Object.keys(PropsAllowlists.NATIVE_THREAD_PROPS_WHITELIST)
+import { SHOULD_BE_USE_WEB } from './common';
+import type { ShadowNodeWrapper } from './commonTypes';
+import {
+  executeOnUIRuntimeSync,
+  jsiRegisterNativePropNamesForComponentName,
+} from './core';
+
+export function addWhitelistedNativeProps(): void {
+  console.warn(
+    '[Reanimated] `addWhitelistedNativeProps` is no longer necessary in Reanimated 4 and will be removed in next version. Please remove this call from your code.'
```

Comment:
```suggestion
  logger.warn(
    '`addWhitelistedNativeProps` is no longer necessary in Reanimated 4 and will be removed in next version. Please remove this call from your code.'
```

I know that we now export `logger` from the `react-native-worklets` package which is not valid as it adds the `[Worklets]` to all logs but I think we should refactor it later, since all logs in reanimated are now handled in this way and it would be easier to keep things consistent in order not to forget to change these logs in the future.